src/script/script.h

```diff
@@ -583,4 +595,15 @@ struct CScriptWitness
     std::string ToString() const;
 };
 
+class CReserveScript
```

Comment:
This was removed in 1b46a4889f894d63062454fb9f4c2e88689a9d75. You've added it back with a bad rebase.